helper/debug.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const _ = require('lodash');
+
+class Debug {
+  constructor(moduleName){
+    this.name = moduleName || 'unnamed module';
+  }
```

Comment:
can't do that because the `req` object is different for every call to push. the constructor happens at startup and the same object is used for all debugging in a given module.